hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/operation/VersionResponsePair.java

```diff
@@ -28,6 +28,7 @@
  * Contains response and partition version for update operations on replicated map.
  */
 public class VersionResponsePair implements DataSerializable, IdentifiedDataSerializable {
```

Comment:
This interface only needs to implement IdentifiedDataserializable, since that extends DataSerializable.